prog/prog.go

```diff
@@ -19,7 +19,8 @@ type Prog struct {
 // IMPORTANT: keep the exact values of "key" tag for existing props unchanged,
 // otherwise the backwards compatibility would be broken.
 type CallProps struct {
-	FailNth int `key:"fail_nth"`
+	FailNth  int  `key:"fail_nth"`
+	Detached bool `key:"detached"`
```

Comment:
Is "async" a better name? I don't have a strong option either way, but it's our last chance to change it.